src/App.tsx

```diff
@@ -1,26 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, {useEffect, useState} from 'react';
```

Comment:
![image](https://user-images.githubusercontent.com/15168701/108961086-e899fb00-7687-11eb-984b-0651fca9fa2f.png)

Reply:
![image](https://user-images.githubusercontent.com/15168701/108963274-f1d89700-768a-11eb-9ddf-67ecd692fe95.png)
need to translate to english